pyphare/pyphare/simulator/simulator.py

```diff
@@ -294,12 +294,13 @@ def _log_to_file(self):
         Support keys:
             RANK_FILES - logfile per rank
             DATETIME_FILES - logfile with starting datetime timestamp per rank
-            NONE - no logging files, display to cout
+            CLI  - no logging files, display to cout
+            NULL - no logging files, no cout
         """
-
-        if "PHARE_LOG" not in os.environ:
-            os.environ["PHARE_LOG"] = "RANK_FILES"
         from pyphare.cpp import cpp_lib
 
-        if os.environ["PHARE_LOG"] != "NONE" and cpp_lib().mpi_rank() == 0:
+        logging = os.environ["PHARE_LOG"] = os.environ.get("PHARE_LOG", "RANK_FILES")
```

Comment:
possible we should default to no files, or maybe if there's more than 1000 ranks or something